src/static/js/app/survey.js

```diff
@@ -0,0 +1,122 @@
+import 'fetch';
+import React from 'react';
+
+// Use `jspm bundle-sfx ./src/static/js/app/survey ./src/static/js/out.js` to build this bad boy up into
+// a lightning-fast executable. Run from the base of the project!
+
+class SurveyAnswerGroup extends React.Component {
+  render() {
+    let answerName = "choice" + this.props.questionNumber;
+    let alwaysSelected = this.props.questionAnswer == 5 ? 'checked' : '';
```

Comment:
This isn't used yet, I need to figure out a better way to do this. What I need to be able to do is take the answer they already put in and use it to fill out the survey for them, basically. Right now it's not working so well, ha.